LinkedList/MyLinkedList.cpp

```diff
@@ -0,0 +1,98 @@
+#include "MyLinkedList.h"
+#include <corecrt_malloc.h>
+#include <iostream>
+
+Node* head = nullptr;

```

Comment:
"head" must be member of List. How do u think this code will work if u create 2 or 3 lists?